checkout-ui-custom/src/partials/Deliver/AddressForm.js

```diff
@@ -186,12 +186,26 @@ const AddressForm = () => {
   <form id="bash--address-form" method="post">
     ${formFields}
 
+    <a
```

Comment:
please pull the latest develop, these changes should not be there. i reverted this. @NosiphoManyase